include/admin.jsonnet

```diff
@@ -85,7 +85,16 @@ local common = import "common.jsonnet";
                       {
                         "name": "CONFIGURATION_SERVICE_PORT",
                         "value": "5672"
-                      }]),
+                      },
+                      {
+                        "name": "API_SERVER_HOSTNAME",
```

Comment:
I prefer MESSAGING_ADMIN_SERVICE_HOST, but I would be fine with your suggestion .